src/main/java/ui/components/FilterTextField.java

```diff
@@ -264,6 +273,15 @@ public FilterTextField setOnConfirm(Function<String, String> confirm) {
     }
 
     /**
+     * Sets the 'onHelp' callback, which will be called when the key combination defined by
+     * KeyboardShortcuts.SHOW_HELP is pressed with this object in focus.
+     */
+    public FilterTextField setOnHelpKeyCombination(Runnable onHelp) {
```

Comment:
Don't think we need to talk about how the method is used in here. The method name can be a bit shorter also, since the help action is not necessarily triggered by a key combination -- that just happens to be the current implementation.